docs(*): move admin port to DPP resource (#667)
kumahq/kuma-website: docs/docs/dev/networking/networking.md

When we start a data-plane via `kuma-dp` we expect all the inbound and outbound service traffic to go through it. The inbound and outbound ports are defined in the dataplane specification when running in universal mode, while on Kubernetes the service-to-service traffic always runs on port `15001`.

In addition to the service traffic ports, the data-plane automatically also opens the `envoy` [administration interface](https://www.envoyproxy.io/docs/envoy/latest/operations/admin) listener on the `127.0.0.1:9901`.

> More information on Envoy Admin port configuration could be found [here](../documentation/dps-and-data-model/#envoy).

Reviewer: ```suggestion
Check the [dpp documentation](../documentation/dps-and-data-model/#envoy) for more on Envoy Admin port.
```
